### PR TITLE
add installation instructions to README

### LINKED ISSUE #33: Document how to install the oria-operator on a cluster
**Goal**: Provide documentation on how to install oria-operator on a cluster

**Acceptance Criteria**:
- Installation document in the oria-operator repo (README or in a docs folder)

### PR DESCRIPTION
**Pull request by everettraven**
**Description of the change**:
- Add installation instructions to the README
    - Adds instructions for installing from latest release, a specific release, or from the current `master` branch

**Motivation**:
- fixes #33

### CODE REVIEW

**Comment by everettraven** [comment]
Blocked on #65 
/hold

File: README.md [hunk 1]
```diff
@@ -63,6 +63,22 @@ The reconciliation process will verify the below steps:
 1. It will look for `ScopeTemplate` that `ScopeInstance` is referencing. if it is not referencing then throw an error with the appropriate message.
 2. If it is referencing and if the `namespaces` array is empty, a single `ClusterRoleBinding` will be created. Otherwise, a `RoleBinding` will be created in each of the `namespaces`. These resources will include an owner reference to the `ScopeInstance` CR.
 
+## Installation
+To install the latest release of `oria-operator`, run:
+```
+kubectl apply -f https://github.com/operator-framework/oria-operator/releases/latest/download/oria-operator.yaml
```
Comment (laxmikantbpandhare) [inline README.md:69]: Getting `not found` for this after release it will generate right?

**Review by laxmikantbpandhare — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (everettraven) [reply README.md:69]: Yeah it should be generated once a release is created. We currently don't have any releases so it won't work for now.

Comment (everettraven) [reply README.md:69]: If you want to see that it works properly you can replace `operator-framework` with `everettraven` as I created a test release on my fork: https://github.com/everettraven/oria-operator/releases/tag/v0.0.3

Comment (laxmikantbpandhare) [reply README.md:69]: got it. I will try these instructions afterwards. Looks good to me.

**Comment by everettraven** [comment]
I guess we should probably hold this on an initial release.

/hold

File: README.md [hunk 2]
```diff
@@ -63,6 +63,17 @@ The reconciliation process will verify the below steps:
...
+```
+
+To install a specific release of `oria-operator`, run: 
+```
+ORIA_VERSION=vX.Y.Z; kubectl apply -f https://github.com/operator-framework/oria-operator/releases/download/$ORIA_VERSION/oria-operator.yaml
```
Comment (everettraven) [inline README.md:74]: What are thoughts on this just being:
```suggestion
kubectl apply -f https://github.com/operator-framework/oria-operator/releases/download/vX.Y.Z/oria-operator.yaml
```

**Review by jmrodri — APPROVED** [review]
/lgtm

**Pull request merged**